### PR TITLE
[#56] Improves message about files 360 cove can handle

### PR DESCRIPTION
**Pull request by caprenter**
Also adds tests
Has a commented out HTML line for cvs files that we are working on

### CODE REVIEW

File: cove/templates/base_360.html [hunk 1]
```diff
@@ -14,6 +14,17 @@
...
+  
+  <h3><small>{% blocktrans %}Formats{% endblocktrans %}</small></h3>
+  <p>{% blocktrans %}
+  The application accepts data in some of the formats given in the <a href="http://docs.threesixtygiving.org/publish/">360 Giving Data Standard publishing guidence</a> under <a href="http://docs.threesixtygiving.org/publish/#toc1">Choose your template</a>.
+  <br>Acceptable files are: {% endblocktrans %}</p>
+  <ul>
+    <li>{% blocktrans %}Summary Spreadsheet - <a href="http://docs.threesixtygiving.org/assets/standard/schema/summary-table/360-giving-schema-titles.xlsx">Excel</a>{% endblocktrans %}</li>
+    <!--<li>{% blocktrans %}Summary Spreadsheet - <a href="http://docs.threesixtygiving.org/assets/standard/schema/summary-table/360-giving-schema-titles.csv/Activity.csv">CSV</a>{% endblocktrans %}</li>-->
+    <li>{% blocktrans %}JSON built to the <a href="http://docs.threesixtygiving.org/docs/#json-schema">360 Giving Data Standard JSON schema</a>{% endblocktrans %}</li>
+    <li>{% blocktrans %}Multi-table data package - <a href="http://docs.threesixtygiving.org/assets/standard/schema/multi-table/360-giving-schema-fields.xlsx">Excel</a>{% endblocktrans %}</li>
```
Comment (Bjwebb) [inline cove/templates/base_360.html]: "Summary Spreadsheet", "Multi-table data package" -> not sure they are the best terms

("data package" is already very much in use for a certain Open Knowledge thing, "summary" suggests the data has been summarized in some way.)

**Comment by Bjwebb** [comment]
Now that I've pushed my CSV work up, this doesn't merge automatically, and some of it's no longer correct (as CSVs are now supported).

@caprenter Do you want to have a go at rebasing this against the new master?

File: cove/templates/base_360.html (hunk 1 above)
Comment (caprenter) [inline cove/templates/base_360.html]: This is based on and replicates the text used on the 360 documentation which is why I have chosen to use them.

Comment (Bjwebb) [inline cove/templates/base_360.html]: Makes sense, thanks.

**Pull request merged**